From: Carlos Maddela <e7appew@gmail.com>
Date: Thu, 10 Jan 2019 04:34:24 +1100
Subject: Fix program termination by SIGILL signal,
 by removing sse4 instructions.

Description: Fix program termination by SIGILL signal,
 by removing sse4 instructions.
Author: Massimiliano Torromeo
Origin: vendor, https://git.archlinux.org/svntogit/community.git/tree/trunk/no-sse4.patch?h=packages/rmlint
Forwarded: not-needed
Last-Update: 2019-01-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 SConstruct | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/SConstruct b/SConstruct
index 7370585..d542e1d 100755
--- a/SConstruct
+++ b/SConstruct
@@ -662,10 +662,7 @@ if conf.env['IS_CYGWIN']:
 else:
     conf.env.Append(CCFLAGS=['-fPIC'])
 
-# check _mm_crc32_u64 (SSE4.2) support:
-conf.check_mm_crc32_u64()
-if conf.env['HAVE_MM_CRC32_U64']:
-    conf.env.Append(CCFLAGS=['-msse4.2'])
+conf.env['HAVE_MM_CRC32_U64'] = False
 
 if 'clang' in os.path.basename(conf.env['CC']):
     conf.env.Append(CCFLAGS=['-fcolor-diagnostics'])  # Colored warnings
